### PR TITLE
Homepage en savoir plus

### PR DESCRIPTION
**Pull request by thalice**

### CODE REVIEW

File: frontend/src/components/EnSavoirPlus.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function EnSavoirPlus() {
```
Comment (BastienSaulnier) [inline frontend/src/components/EnSavoirPlus.jsx]: Please rename your component to "LearnMore" for uniform language use

File: frontend/src/components/styles/EnSavoirPlus.scss [hunk 2]
```diff
@@ -6,8 +8,13 @@
...
-  border: black solid 2px;
-  border-radius: 10px;
+  border: $mainBrandColor solid 2px;
+  border-radius: 3px;
   height: 100%;
   width: 100%;
+  text-align: center;
+}
+
+#lirePlus {
```
Comment (BastienSaulnier) [inline frontend/src/components/styles/EnSavoirPlus.scss]: Please use English selector Names

File: frontend/src/components/EnSavoirPlus.jsx [hunk 3]
```diff
@@ -1,5 +1,17 @@
 import React from "react";
+import "./styles/EnSavoirPlus.scss";
```
Comment (BastienSaulnier) [inline frontend/src/components/EnSavoirPlus.jsx]: Please rename your scss file to LearnMore.scss

File: frontend/src/components/styles/EnSavoirPlus.scss [hunk 4]
```diff
@@ -12,6 +12,9 @@
   border-radius: 2px;
   height: 100%;
   width: 300px;
+  padding: 20px;
+  border: 1px solid $mainBrandColor;
+  text-align: justify;
 }
 
 #titre {
```
Comment (BastienSaulnier) [inline frontend/src/components/styles/EnSavoirPlus.scss]: Please use English selector names

**Review by BastienSaulnier — CHANGES_REQUESTED** [review]
Please use english almost everywhere exept in JSX content ( appart classnames & ids )

File: frontend/src/components/EnSavoirPlus.jsx (hunk 1 above)
Comment (thalice) [reply frontend/src/components/EnSavoirPlus.jsx]: Checked

File: frontend/src/components/styles/EnSavoirPlus.scss (hunk 2 above)
Comment (thalice) [reply frontend/src/components/styles/EnSavoirPlus.scss]: Checked

File: frontend/src/components/EnSavoirPlus.jsx (hunk 3 above)
Comment (thalice) [reply frontend/src/components/EnSavoirPlus.jsx]: Checked

File: frontend/src/components/styles/EnSavoirPlus.scss (hunk 4 above)
Comment (thalice) [reply frontend/src/components/styles/EnSavoirPlus.scss]: checked

**Comment by BastienSaulnier** [comment]
Everything looking ok for me now !

**Review by BastienSaulnier — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Branch name in FR... Other than that, No Evil Seen ! ;-)

**Review by BastienSaulnier — DISMISSED** [review]
Appart FR branch name, conflicts resolved.

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**